node/clients/storefrontPermissions.ts

```diff
@@ -40,6 +41,22 @@ export default class StorefrontPermissions extends AppGraphQLClient {
     })
   }
 
+  public getOrganizationsByEmailPaginated = async (
```

Comment:
Do we need this function here and on the storefront-permissions app? ([PR](https://github.com/vtex-apps/storefront-permissions/pull/157#pullrequestreview-2280369421))

Can't we query Master Data directly from this app and avoid additional call to another app?

Reply:
@enzomerca I'm following the same approach of the getOrganizationsByEmail query

Reply:
Yes, but we are removing the cyclic dependencies, and this is one of them, and thinking about the domain and context, the storefront shouldn't know about the organization.